src/plugins/dashboard/public/dashboard_app/dashboard_app.tsx

```diff
@@ -68,9 +73,16 @@ export function DashboardApp({
   history,
 }: DashboardAppProps) {
   const [showNoDataPage, setShowNoDataPage] = useState<boolean>(false);
+  const [redirectToExpandedPanel, setRedirectToExpandedPanel] = useState<string | undefined>();
 
   useMount(() => {
-    (async () => setShowNoDataPage(await isDashboardAppInNoDataState()))();
+    (async () => {
+      setShowNoDataPage(await isDashboardAppInNoDataState());
+      const state: DashboardContainerInput | null = kbnUrlStateStorage.get(
+        DASHBOARD_STATE_STORAGE_KEY
+      );
+      if (state) setRedirectToExpandedPanel(state?.expandedPanelId);
```

Comment:
Can you explain this a bit? I'm curious why this was added

Reply:
Good call, I have added a comment in the code to make it clearer to others why this was added. I'm open to advice on how it should be phrased, `// if given a shareable URL with an expanded panel, this is needed to open to the expanded panel when the dashboard loads` 

Since the shareable URL will have the _a state in its url, this is checking for the expanded panel ID in the URL.  When opened in a new tab, these lines of code are needed to show the expanded panel when the dashboard loads.